packages/graphql-tag-pluck/package.json

```diff
@@ -58,10 +58,12 @@
     "@graphql-tools/utils": "^10.8.6",
     "tslib": "^2.4.0"
   },
+  "optionalDependencies": {
```

Comment:
Is this really needed because this will make most of package managers to install this package?

Reply:
This package *does* depend on `@vue/compiler-sfc` at runtime so `devDependencies` feels ill-suited to me. I'd err on either `optionalDependencies` or `peerDependencies`. What's your preference? I'm happy to update this however you'd like.

Reply:
Fair enough, it's a good point to separate fixing the bug from changing the dependency management. Updated.